### ISSUE TITLE
when color prop set cursor does not work

### ISSUE THREAD

**Issue by nadimtuhin** [casesandberg/react-color#35]
this code does not work

```javascript
import React 'react';
import ReactColor from 'react-color';


class ColorPicker extends React.Component {

  constructor() {
    super();
    this.state = {
      value: '#fff'
    };

    this.handleChangeComplete = this.handleChangeComplete.bind(this);
  }

  handleChangeComplete(color) {
    this.setState({value: color.hex});
  }

  render() {
    return <ReactColor color={ this.state.value }
                       type="sketch"
                       onChangeComplete={ this.handleChangeComplete } />;
  }
}

export default ColorPicker;
```

**Comment by adeelraza** [comment]
Duplicate of #34 - the problem is not setting the default color but it breaks if the color is #fff or #000

**Comment by casesandberg** [comment]
This was fixed in `react-color@1.0.6`
